src/jail_rules_p.h

```diff
@@ -40,7 +40,7 @@
 #include "jail_types_p.h"
 #include "jail_rules.h"
 
-#define DEFAULT_PROFILE_SECTION "Sailjail"
+#define DEFAULT_PROFILE_SECTION "X-Sailjail"
```

Comment:
Make it "X-Sailjail" for .desktop files (which is a special case), "Sailjail" by default.

Reply:
Dang. Now I notice this.